src/main/java/nodebox/client/devicehandler/OSCDeviceHandler.java

```diff
@@ -216,25 +216,25 @@ public void actionPerformed(ActionEvent actionEvent) {
         private void startOSC() {
             start();
             if (isRunning())
-            startButton.setText(isRunning() ? "Pause" : "Start");
+            startButton.setText(isRunning() ? PAUSE : START);
             stopButton.setEnabled(oscP5 != null);
         }
 
         private void resumeOSC() {
             resume();
-            startButton.setText(isRunning() ? "Pause" : "Start");
+            startButton.setText(isRunning() ? PAUSE : START);
             stopButton.setEnabled(true);
         }
 
         private void pauseOSC() {
             pause();
-            startButton.setText(isRunning() ? "Resume" : "Start");
+            startButton.setText(isRunning() ? "Resume" : START);
```

Comment:
I know, I assume that's why your tool picked it up. However here for consistency I would refactor it out as well.